game/scripts/npc/abilities/lion_impale.txt

```diff
@@ -46,7 +46,7 @@
       "02"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "duration"                                        "1.4 1.8 2.2 2.6 2.8 3.0"
+        "duration"                                        "1.4 1.8 2.2 2.4 2.5 2.6" //OAA
```

Comment:
I feel like this is an unnecessary nerf.

Reply:
maybe just nerf the manacost so he can't spam this early on and makes it so lions can't just got 3/2/0 and have to pick up an early mana drain to stay effective.

Reply:
Eh no, its justified nerf